common/utils.py

```diff
@@ -13,8 +14,8 @@ def circulate(count: int, current: int, to_end: bool):
     return (count + current + (1 if to_end else -1)) % count
 
 
-def button(k, t):
-    return "[color=orange][[" + k + "]][/color] " + t
+def button(key, caption):
+    return "[color=orange]" + bltutils.lucynest_L + key.lower() + bltutils.lucynest_T + "[/color] " + caption.lower()
```

Comment:
I suggested to use a "colored" function from bltutils, and rename lucynest_* constants to styled_* and maybe make a separate function for adding those "letters" to a string and use it (because we already use such thing for game title and now here, and maybe later will use even more)
And I think here is no need to making string lowercase